agent-inject/agent/annotations.go

```diff
@@ -82,6 +82,24 @@ const (
 
 	// AnnotationAgentInjectToken is the annotation key for injecting the
 	// auto-auth token into the secrets volume (e.g. /vault/secrets/token)
+	// AnnotationAgentInjectTemplateLeftDelim is the key annotation that configures Vault
+	// Agent what left delimiter to use for rendering the secrets.  The name
+	// of the template is any unique string after "vault.hashicorp.com/agent-inject-left-delimiter-",
+	// such as "vault.hashicorp.com/agent-inject-left-delimiter-foobar".  This should map
+	// to the same unique value provided in "vault.hashicorp.com/agent-inject-secret-".
+	// If not provided, a default left delimiter is used as defined by https://www.vaultproject.io/docs/agent/template#left_delimiter
+	AnnotationAgentInjectTemplateLeftDelim = "vault.hashicorp.com/agent-inject-left-delimiter"
```

Comment:
changed the inject-template-left-delimiter from inject-left-delimiter since the default pull request

Reply:
I was also thinking we could shorten it a little more and use something like `agent-template-left-delim` here? I like having "template" in the annotation key since this is a template option.

Reply:
Yep, I'll change that 👍